### DISCUSSION TITLE
Allow line-clamping more than 6 lines

### DISCUSSION THREAD

**Discussion by rassom** [tailwindlabs/tailwindcss#10808] [Ideas]
I have a user interface with a narrow notes section but line-clamping to only 6 lines is not quite enough, IMO:
<img width="499" alt="image" src="https://user-images.githubusercontent.com/594378/225606937-5e98203d-f025-4703-9067-f255f2a1a82b.png">

Could you maybe let users of line-clamp decide how many lines rather than limit to just 6?

Would be much appreciated. Thank you :)

**Comment by wongjn** [comment]
Assuming this is in regards to the `@tailwindcss/line-clamp` package, you can do one of:

- Add extra values to your Tailwind configuration
  ```js
  module.exports = {
    theme: {
      extend: {
        lineClamp: {
          7: '7',
          8: '8',
          9: '9',
          10: '10',
          // Et cetera.
        }
      }
    },
    plugins: [
      require('@tailwindcss/line-clamp'),
    ],
  };
  ```
- Use an [arbitrary value class](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values), like `line-clamp-[20]`.

Which should cover any extra values you'd like.

**Reply by rassom** [reply]
Perfect. Thank you :)
